locales/en-US/app.ftl

```diff
@@ -737,12 +737,26 @@ TrackMemoryGraph--operations-since-the-previous-sample = operations since the pr
 # This is used in the tooltip when the power value uses the Watt unit.
 # Variables:
 #   $value (String) - the power value at this location
-TrackPowerGraph--tooltip-power-watt = Power: <em>{ $value } W</em>
+TrackPower--tooltip-power-watt = { $value } W
+  .label = Power
 
 # This is used in the tooltip when the power value uses the Milliwatt unit.
 # Variables:
 #   $value (String) - the power value at this location
-TrackPowerGraph--tooltip-power-milliwatt = Power: <em>{ $value } mW</em>
+TrackPower--tooltip-power-milliwatt = { $value } mW
+  .label = Power
+
+TrackPower--tooltip-power-used-in-range-watthour = { $value } Wh
+  .label = Power used in range
+
+TrackPower--tooltip-power-used-in-range-milliwatthour = { $value } mWh
+  .label = Power used in range
+
+TrackPower--tooltip-power-used-in-preview-watthour = { $value } Wh
+  .label = Power used in temporary selection
```

Comment:
I don't think temporary is the right choice here, or at least I'm confused on the meaning. Does it refers to the fact that this is the power consumption for a specific time range?

Reply:
In the UI, there's a distinction between the "committed" range, and the preview selection. From the preview selection we can either click "+" to make it the "committed" range (and pass it full screen), or click elsewhere to unselect it. That's what I call the "temporary" selection. But I'm not especially happy about that either.

Our internal term is "preview selection" but I'm not sure this is understandable either.

Reply:
I think using the word "Energy" instead of "Power" for the amount of energy used over a range would be more correct.

Ideas of alternative wordings:
"Energy used in the visible range" & "Energy used in the selected range"
And I'm not sure about the word "visible", I also considered "full", "entire", "current". None of them feels perfect. "committed range" is technically correct but too jargonish.

Reply:
I used `Energy used in the visible range` and `Energy used in the current selection`. Not especially obvious but not bad either. I guess the meaning is well discoverable with testing.